File: font-end/alianza-front/src/app/model/resultDTO.ts
import { ClienteDTO } from "./clienteDTO";

export class ResultDTO {
  msgRespuesta : string;
  lcliente : ClienteDTO[];

  constructor(
    msgRespuesta?:string,
    lcliente?:ClienteDTO[]
  ){

    this.msgRespuesta = msgRespuesta;
    this.lcliente = lcliente;
  }
} 

File: alianza-api/src/main/java/com/alianza/alianzaapi/dto/ResultDTO.java
package com.alianza.alianzaapi.dto;

import java.util.List;

public class ResultDTO {
	
	private String msgRespuesta;
	
	private List<ClienteDTO> lcliente;

	public String getMsgRespuesta() {
		return msgRespuesta;
	}

	public void setMsgRespuesta(String msgRespuesta) {
		this.msgRespuesta = msgRespuesta;
	}

	public List<ClienteDTO> getLcliente() {
		return lcliente;
	}

	public void setLcliente(List<ClienteDTO> lcliente) {
		this.lcliente = lcliente;
	}
	
}
